### ISSUE TITLE
Question: loss going up

### ISSUE THREAD

**Issue by ghost** [huggingface/autotrain-advanced#153]
I fine tune with the below script. Loss initially goes down but then starts going up and generated output is a mess. The same data worked for llama1, so i think is ok. Do i have any errors in the setup below?

I truncated data to 1024 so it's equal to default block size.

!autotrain llm --train \
--project_name out \
--model meta-llama/Llama-2-13b-hf \
--data_path data-1024 \
--use_peft \
--trainer sft \
--learning_rate 3e-4 \
--train_batch_size 8 \
--gradient_accumulation_steps 16 \
--num_train_epochs 3 \
--logging_steps 1 \
--use_int4 \

**Comment by abhishekkrthakur** [comment]
can you try removing the sft trainer and use the default one? you can just remove `--trainer sft`. we are currently investigating the same.

**Comment by mimumo** [comment]
Thanks! Tried it with removed `--trainer sft` arg. However looks like loss still going up...
<img width="766" alt="image" src="https://github.com/huggingface/autotrain-advanced/assets/131650302/6ba4fe7f-cc20-42d9-bf05-f87ba92e5730">
